### ISSUE TITLE
Add README

### ISSUE THREAD

**Issue by LeonardoGentile** [Jawabiscuit/obsidian-modules#1]
Hello, 
I've just wanted to follow up [this discussion](https://github.com/mdelobelle/metadatamenu/discussions/543).
I went to scavenge your github and found out your awesome Obsidian setup:

- https://github.com/Jawabiscuit/.obsidian
- https://github.com/Jawabiscuit/obsidian-templates -> `_templates`
- https://github.com/Jawabiscuit/obsidian-templater-scripts -> `_scripts`
- https://github.com/Jawabiscuit/obsidian-views -> `_views`
- https://github.com/Jawabiscuit/obsidian-modules -> `_modules`
- https://github.com/Jawabiscuit/obsidian-modules -> `_mdm`

For all of these I've activated the `sandbox` branch (where available)  and I finally made it everything works. 

You must know that there is at least one person using your setup and I find it awesome :D 

I'm aware you don't want to be obligated to maintain a public plugin or ecosystem for your setup but may I ask few questions to let folks like me understand how to manage and glue everything together? 
This is just to understand and find inspiration from your efforts and then whoever find it useful can simply fork and go on with their own setup.

So, in short, may I ask few questions and where do you feel it is better suited for that? Should you open a discussion tab or is it fine in here? Mine are not issues but simple questions that's why I think a discussion might be more useful. 

Then from your answer and my understanding of your ideas I might help you setup a README for all your obsidian repos.
